plugins/awos/scripts/status-board.py

```
@@ -0,0 +1,196 @@
+        ids = list(dict.fromkeys(re.findall(r"[A-Z]{3,}-\d+", tline)))
+        ticket = ids[0] + (f" +{len(ids) - 1}" if len(ids) > 1 else "") if ids else ""
+        tm = os.path.join(d, "tasks.md")
+        tasks = 0
+        if os.path.exists(tm):
+            for ln in open(tm, encoding="utf-8", errors="replace"):
+                # atomic tasks only; composite "- [ ] **Slice N**" headers would double-count
+                if re.match(r"\s*- \[[ xX]\]", ln) and not re.match(
+                    r"\s*- \[[ xX]\]\s*\*\*Slice", ln
+                ):
+                    tasks += 1
+        ct = subprocess.run(
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)